Configurations.md

```diff
@@ -2408,6 +2408,27 @@ pub enum Foo {}
 pub enum Foo {}
 ```
 
+## `inline_attribute_width`
+
+Write an item and its attributes on the same line if their combined width is below a threshold
```

Comment:
Typo: `s/attributes/attribute/`, since this only applies when there is only a single attribute.